packages/overlays/src/utils/contain-focus.js

```diff
@@ -101,6 +101,7 @@ export function containFocus(rootElement) {
   function createHelpersDetectingTabDirection() {
     tabDetectionElement = document.createElement('div');
     tabDetectionElement.style.display = 'none';
+    tabDetectionElement.setAttribute('is-tab-detection-element', '');
```

Comment:
Could we prefix with `data-`, so it's w3c validator compliant? => https://validator.w3.org/nu/#textarea
(non data attrs are only allowed on custom elements)

Reply:
Ah true, updated